### ISSUE TITLE
Feature request: support Ansible 2.10

### ISSUE THREAD

**Issue by yan12125** [geerlingguy/ansible-role-swap#14]
Currently this role does not work with Ansible 2.10

```
$ ansible localhost -m import_role -a "name=geerlingguy.swap" --check
[WARNING]: You are running the development version of Ansible. You should only run Ansible from "devel" if you are modifying the Ansible engine,
or trying out features under development. This is a rapidly changing source of code and can become unstable at any point.
ERROR! couldn't resolve module/action 'mount'. This often indicates a misspelling, missing collection, or incorrect module path.

The error appears to be in '/home/yen/.ansible/galaxy-roles/geerlingguy.swap/tasks/main.yml': line 2, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

---
- name: Manage swap file entry in fstab.
  ^ here

```

I managed to get it work by adding the following lines to `~/.ansible/galaxy-roles/geerlingguy.swap/meta/main.yml`

```
collections:
  - ansible.posix
```

I'm not sure if it breaks compatibility with older Ansible versions. If so, maybe a new release with `min_ansible_version: 2.10`?

**Comment by yan12125** [comment]
With ansible 2.10.0b1, this role works just fine. I guess routing for collections in ansible was buggy when I created this ticket.

**Comment by worldofgeese** [comment]
I was just bit by this and would say this is a bug, given I'm running the latest release Ansible available from Nixpkgs: 2.10.

@geerlingguy

**Comment by yan12125** [comment]
This role appears to work for me with ansible 2.10
```
$ ansible-galaxy role list | grep geerlingguy.swap
- geerlingguy.swap, 1.0.2
$ ansible localhost -m import_role -a "name=geerlingguy.swap" --check
localhost | CHANGED => {
    "changed": true,
    "dump": "0",
    "fstab": "/etc/fstab",
    "fstype": "swap",
    "name": "none",
    "opts": "sw",
    "passno": "0",
    "src": "/swapfile"
}
localhost | SKIPPED
localhost | SUCCESS => {
    "changed": false,
    "include": "enable.yml",
    "include_args": {}
}
localhost | CHANGED | rc=0 >>
Command would have run if not in check modeCommand would have run if not in check mode
localhost | FAILED! => {
    "changed": false,
    "msg": "file (/swapfile) is absent, cannot continue",
    "path": "/swapfile",
    "state": "absent"
}
```
Maybe it is a Nix-specific issue?
